exonum/src/blockchain/builder.rs

```diff
@@ -89,6 +88,17 @@ impl BlockchainBuilder {
         self.with_additional_runtime(runtime)
     }
 
+    pub fn with_external_runtimes(
```

Comment:
Now we have 4 (!) methods for builder that are used to add Runtime:
`with_external_runtimes`, `with_additional_runtime`, `with_default_runtime` and `with_rust_runtime`. Don't you think it's somewhat complicated?

Reply:
Yeah, it should be refined too.